tests/kernel/install_ltp.pm

```diff
@@ -165,10 +166,29 @@ sub install_from_repo {
     assert_script_run q(find ${LTPROOT:-/opt/ltp}/testcases/bin/openposix/conformance/interfaces/ -name '*.run-test' > ~/openposix-test-list-) . $tag;
 }
 
+sub add_custom_grub_entries {
+    my $grub_param = shift;
+    return unless defined($grub_param);
+    assert_script_run('cp /boot/grub2/grub.cfg grub.cfg.old');
+    upload_logs('grub.cfg.old', failok => 1);
+
+    assert_script_run('cp /etc/grub.d/40_custom 40_custom.tmp');
+    my $distro = (is_sle() ? "SLES" : "openSUSE") . " " . get_required_var('VERSION');
+
+    my $cmd = q(grub2-mkconfig | sed -ne '/\/etc\/grub.d\/10_linux/,\/etc\/grub.d\/10_linux/p' | sed -e 's/\\(\/boot\/vmlinuz.*\\)/\1);
+    $cmd .= " $grub_param/' -e 's/\\(menuentry .$distro\\)/\\1 ($grub_param)/' >> 40_custom.tmp";
+    assert_script_run($cmd);
+    assert_script_run('cp 40_custom.tmp /etc/grub.d/40_custom');
+    upload_logs('/etc/grub.d/40_custom', failok => 1);
+    assert_script_run('grub2-mkconfig -o /boot/grub2/grub.cfg');
+    upload_logs('/boot/grub2/grub.cfg', failok => 1);
+}
+
 sub run {
-    my $self     = shift;
-    my $inst_ltp = get_var 'INSTALL_LTP';
-    my $tag      = get_var('LTP_RUNTEST_TAG') || get_var('VERSION') . '-' . get_var('BUILD');
+    my $self       = shift;
+    my $inst_ltp   = get_var 'INSTALL_LTP';
+    my $tag        = get_var('LTP_RUNTEST_TAG') || get_var('VERSION') . '-' . get_var('BUILD');
+    my $grub_param = get_var('LTP_GRUB_PARAM') || 'ima_policy=tcb';
```

Comment:
maybe just call this GRUB_PARAM and add a new test module which adds boot options. I imagine this might be useful for a lot of tests besides LTP

Reply:
Well, before I created grub_utils and send pull request https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5030, but it wasn't really useful. This is different code, maybe reusable for somebody => ok, I'll try to add it into `bootloader_setup.pm` (hope it wont be wasted time again).